.tekton/tasks/update-dependencies.yaml

```diff
@@ -36,17 +36,27 @@ spec:
         && echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
         && apt update \
         && apt install gh -y
-
+        apt install jq -y
+        
         gh auth login --with-token
 
         git ls-remote --exit-code --heads origin currency-bot >/dev/null 2>&1
-        EXIT_CODE=$?
+        BRANCH_EXISTS=$?
 
-        if [[ $EXIT_CODE != '0' ]]; then          
+        if [[ $BRANCH_EXISTS != 0 ]]; then
           echo "Running patch and minor updates..."
           MAJOR_UPDATES_MODE=false BRANCH=currency-bot node bin/currency/update-currencies.js    
         else
-          echo "Skipping patch/minor updates. Branch exists."
+          echo "currency-bot branch exists. Checking for open pull requests.."
```

Comment:
issue: The logic will only work for the major updates.

See line 63
> MAJOR_UPDATES_MODE=true BRANCH=currency-bot-major node bin/currency/update-currencies.js

The logic needs to be added before line 46.

Reply:
Good catch ❤️